package.json

```diff
@@ -47,6 +47,8 @@
     "docker:reset": "./bin/reset-all",
     "docker:start": "docker-compose up",
     "docker:stop": "docker-compose down",
+    "docker:dbs:start": "docker-compose -f 'docker-compose.yml' up",
+    "docker:dbs:stop": "docker-compose -f 'docker-compose.yml' down",
```

Comment:
Calling `docker-compose` with no `-f` argument loads both `docker-compose.yml` and `docker-compose.override.yml`. Calling it with `-f 'docker-compose.yml'` prevents the override file (and the backend/frontend/worker services) from being included.